server/constants/tables.json

```diff
@@ -1,5 +1,9 @@
 {
     "appointment": "Appointment",
     "student": "Student", 
-    "worker": "Service_Worker"
+    "worker": "Service_Worker", 
+    "timeslot": "Timeslot", 
+    "workerTimeslot": "Worker_Timeslot",
+    "school": "School", 
+    "schoolWorker": "School_Worker"     
```

Comment:
Hi Peter,
I may have missed it, but where is `schoolWorker` currently being used? It seems like the table has not been created yet and is not needed for this task. If possible, I would recommend removing line 8 to prevent future confusion.

Reply:
It has been removed